chapters/7-publication.tex

```diff
@@ -245,106 +243,85 @@ \subsection{Getting started with {\LaTeX} as a team}
 They vary dramatically in their ability to integrate
 with file systems where you store your code and outputs,
 and so you will need to practice an integrated workflow depending what is available to you.
-Despite this, we believe that with minimal learning and workflow adjustments,
-cloud-based implementations are often the easiest way to allow coauthors to write and edit in {\LaTeX},
-so long as you make sure you are available to troubleshoot minor issues like these.
+Some teams adopt cloud-based tools as a permanent solution,
+though others shift to integrated editing and compiling tools such as {\LaTeX} and Atom. 
```

Comment:
```suggestion
though our recommendation is to eventually shift to 
local editing and compiling using tools such as TexStudio and code editors like Atom. 
```
Some reasons for these edits:
1. Make our recommendation to be to eventually no use cloud based latex setups
2. LaTeX is not a tool. 
3. And I think the alternative to "cloud-based" is not "integrated" but local.
4. I think TexStudio should be mentioned, and Atom as an example of code editors.

Reply:
Done in ab95abe3993028c53482b897e795d04407f49531